packages/web/src/scss/components/Spinner/index.html

```diff
@@ -1,35 +1,70 @@
 {{#> layout/plain }}
 
 <section class="docs-Section">
-  <h2 class="docs-Heading">Spinner</h2>
 
-  <div>
+  <h2 class="docs-Heading">Default</h2>
+  
+  <div class="docs-Stack docs-Stack--start">
+
     <svg width="24" height="24" class="animation-spin-clockwise">
```

Comment:
we are probably missing aria-hidden=true everywhere, can you please add it? It is a default in the Icon component.

Reply:
we agreed there will be separate pull request adding this everywhere